src/notte/agents/main.py

```diff
@@ -56,3 +57,41 @@ async def async_run(self, task: str, url: str | None = None) -> AgentResponse:
     def run(self, task: str, url: str | None = None) -> AgentResponse:
         agent = self.create_agent()
         return asyncio.run(agent.run(task, url=url))
+
+    # Add to the relevant section that handles tasks
+    async def handle_arxiv_task(self, task: str, url: str) -> str:
```

Comment:
`handle_arxiv_task` should be moved to `examples/axiv.py` (this is a arxiv specific use-case